### PR TITLE
Add success example with body copy to the guidance

### PR DESCRIPTION
**Pull request by CharlotteDowns**
I've 'hopefully' added a two line success example to the guidance to answer [this open issue](https://github.com/alphagov/govuk-design-system/issues/1407)

### CODE REVIEW

**Comment by CharlotteDowns** [comment]
Not sure how I've made changes to the cookie banner index.njk file or the package-lock.json file 🤔

File: src/components/notification-banner/success/code.njk [hunk 1]
```diff
@@ -5,7 +5,14 @@ layout: layout-example.njk
 
 {% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
 
+{% set html %}
+  <p class="govuk-notification-banner__heading">
+    Training outcome recorded and trainee withdrawn
+  </p>
+  <p>Contact <a class="govuk-notification-banner__link" href="#">becomingateacher@digital.education.gov.uk</a> if you think there's a problem.</p>
```
Comment (edwardhorsford) [inline src/components/notification-banner/success/code.njk:12]: Could we use a fake email address please? this is a live one used by our service.

Comment (edwardhorsford) [inline src/components/notification-banner/success/code.njk:12]:
```suggestion
  <p>Contact <a class="govuk-notification-banner__link" href="#">becomingateacher@digital.education.gov.uk</a> if you think there’s a problem.</p>
```

File: src/components/notification-banner/success/index.njk [hunk 2]
```diff
@@ -5,8 +5,15 @@ layout: layout-example.njk
 
 {% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
 
+{% set html %}
+  <p class="govuk-notification-banner__heading">
+    Training outcome recorded and trainee withdrawn
+  </p>
+  <p>Contact <a class="govuk-notification-banner__link" href="#">becomingateacher@digital.education.gov.uk</a> if you think there's a problem.</p>
```
Comment (edwardhorsford) [inline src/components/notification-banner/success/index.njk:12]:
```suggestion
  <p>Contact <a class="govuk-notification-banner__link" href="#">becomingateacher@digital.education.gov.uk</a> if you think there’s a problem.</p>
```

File: src/components/notification-banner/success/code.njk (hunk 1 above)
Comment (CharlotteDowns) [reply src/components/notification-banner/success/code.njk:12]: Hi Ed, apologies for that mistake, I'll make the change and thank you for flagging :).

**Review by vanitabarrett — APPROVED** [review]

**Pull request merged**